app/apps/visits/models.py

```diff
@@ -60,7 +60,7 @@ def __get_event_values__(self):
 
     def create_from_top(self, data):
         try:
-            case = Case.objects.get(identification=data["case_identification"])
+            case = Case.objects.get(id=data["case_identification"])
```

Comment:
Blijft dit nog werken? De BWV identifications zijn strings (11111_01 bijvoorbeeld) Is dat compatible met een UUIDField in Django?